src/containers/channelDetail/ChannelDetailContainer.jsx

```diff
@@ -14,13 +14,17 @@ import { ALL_STATUSES } from "../../constants/uiConstants";
 
 class ChannelDetailContainer extends Component {
   async componentDidMount() {
-    if (this.props.selectedSuggestion) {
+    const selectedSuggestion = this.getSelectedSuggestion();
+    if (selectedSuggestion) {
+      if (this.props.selectedSuggestion) {
+        sessionStorage.setItem("tempSelectedSuggestion", JSON.stringify(this.props.selectedSuggestion));
+      }
       const actualDate = new Date();
       let oneMonthBefore = new Date().setMonth(actualDate.getMonth() - 1);
       const oneMonthBeforeDate = new Date(oneMonthBefore);
       let channelData = await getChannel(
-        this.props.selectedSuggestion.tokenAddress,
-        this.props.selectedSuggestion.partnerAddress
+        this.getSelectedSuggestion().tokenAddress,
+          this.getSelectedSuggestion().partnerAddress
```

Comment:
looks like there's some extra whitespace here

Reply:
I will fix this in the other PR that conflicts with this one since the other one has code style fixed.